packages/account/src/Assets/ic-poi-name-dob-example.svg

```diff
@@ -0,0 +1 @@
lterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.06 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_1211_60959"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.1 0"/><feBlend in2="effect1_dropShadow_1211_60959" result="effect2_dropShadow_1211_60959"/><feBlend in="SourceGraphic" in2="effect2_dropShadow_1211_60959" result="shape"/></filter><filter id="k" x="175" y="86.5" width="106" height="30" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.06 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_1211_60959"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.1 0"/><feBlend in2="effect1_dropShadow_1211_60959" result="effect2_dropShadow_1211_60959"/><feBlend in="SourceGraphic" in2="effect2_dropShadow_1211_60959" result="shape"/></filter><filter id="l" x="175" y="118.5" width="109" height="30" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.06 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_1211_60959"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix values="0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0 0.054902 0 0 0 0.1 0"/><feBlend in2="effect1_dropShadow_1211_60959" result="effect2_dropShadow_1211_60959"/><feBlend in="SourceGraphic" in2="effect2_dropShadow_1211_60959" result="shape"/></filter><clipPath id="a"><rect x="25" y=".5" width="218" height="131" rx="8" fill="#fff"/></clipPath><clipPath id="e"><path fill="#fff" transform="rotate(-55.14 120.52 -157.66)" d="M0 0h12v36H0z"/></clipPath><clipPath id="f"><path fill="#fff" transform="rotate(44.745 5.112 190.16)" d="M0 0h12v36H0z"/></clipPath></defs></svg>
```

Comment:
can we run SVGO on this?

Reply:
sure Matin, it was run already. will repeat it again just in case